### ISSUE TITLE
Feature Request: Health Check Endpoint

### ISSUE THREAD

**Issue by Chrisissorry** [orocrm/platform#692]
It is best practice to offer a health check endpoint which can be checked by tools like loadbalancers and orchestration tools like Kubernetes, Swarm or rancher to see whether the instance of the system is still healthy and can respond to requests. 

What we currently see is that it is not possible to use any platform endpoint for traditional health checks. In our scenario, we spin up docker containers running OroPlatform. Until the installation is finished we are not able to call any endpoint, e.g. the login URL, because the system will break immediately because of cache inconsistencies. 

Such a health check should include basic checks like:

* Is the database reachable
* Are all core components reachable and working

A good example is the gitlab health endpoint which is secured via token and offers several output formats: https://docs.gitlab.com/ce/user/admin_area/monitoring/health_check.html

**Comment by sadortun** [comment]
@Chrisissorry I have a bundle for you that does exactly this. And it's extendable. I'll try to OSS it this week.

**Comment by Chrisissorry** [comment]
@sadortun Sounds awesome! Looking forward

**Comment by RocKordier** [comment]
Hey @sadortun 
one year ago 😬  Any updates here? That would be awesome.
Did you make a release yet or could we (@Baachi, @digedag ) assist you?

Eric

**Comment by DimaSoroka** [comment]
hi @RocKordier, we are working on this at this moment, implementation will be done based on https://github.com/liip/LiipMonitorBundle

**Comment by sadortun** [comment]
@Chrisissorry yes, we are using it but it not released yet. I Think @DimaSoroka  solution will be better.
